src/main/java/edu/ucsb/cs156/example/controllers/ApiController.java

```diff
@@ -2,13 +2,19 @@
 
 import edu.ucsb.cs156.example.entities.User;
 import edu.ucsb.cs156.example.models.CurrentUser;
+import edu.ucsb.cs156.example.models.SystemInfo;
 import edu.ucsb.cs156.example.services.CurrentUserService;
+import edu.ucsb.cs156.example.services.SystemInfoService;
+
 import org.springframework.beans.factory.annotation.Autowired;
 
 public abstract class ApiController {
   @Autowired
   private CurrentUserService currentUserService;
 
+  @Autowired
+  private SystemInfoService systemInfoService;
```

Comment:
Do we want to add this as a dependency to all API controllers? I think it makes more sense to autowire this only in the SystemInfoController.